src/bin.ts

```diff
@@ -28,16 +28,16 @@ function printHelp() {
 Usage:
 ${
     prettyPrintRow([
-      ["jsr add @std/log", 'Install the "@std/log" package from jsr.io'],
-      ["jsr remove @std/log", 'Remove the "@std/log" package from the project'],
+      ["jsr add @std/log", 'Install the "@std/log" package from jsr.io.'],
+      ["jsr remove @std/log", 'Remove the "@std/log" package from the project.'],
```

Comment:
Personal suggestion, feel free to ignore.

```suggestion
      ["jsr add @std/log", 'Install the package @std/log.'],
      ["jsr remove @std/log", 'Remove the package @std/log from the project.'],
```